kakaemma/tuzimbe
# 183020402 capture people's names working on the site

#### What does this PR do?
This allows for setting up the project
#### Description of Task to be completed?
- register a user
#### How should this be manually tested?
- Pull and checkout to this branch
- start your virtualenv
- run pip install -r requirements.txt
- Create the database and the databse user
- grant the user access to the databse
- copy the contents of.env.example.com to .env
- add the database connection details to the .env file
- run python run.py in the project root folder
- You should be able to access the registration endpoint at `http://127.0.0.1:5000/api/v1/register`
-  add ```{
    "name":"some_name",
    "user_type":"worker"
}```
- the user should be registered
#### What are the relevant pivotal tracker stories?
[#183020402](https://www.pivotaltracker.com/story/show/183020402)
#### Implementation Plan link(s)
N/A
#### Any background context you want to add?
N/A
#### Screenshots
N/A
